### ISSUE TITLE
Sift force-upgrades play-services to 11.0.0

### ISSUE THREAD

**Issue by edenman** [SiftScience/sift-android#2]
Here's what happens when I try to upgrade to sift-android 0.9.2:
<img width="1146" alt="screen shot 2017-06-13 at 4 23 04 pm" src="https://user-images.githubusercontent.com/1063557/27108950-d6b5787c-5054-11e7-983a-87bad7cae2fc.png">
For reference, this is what I'm currently pinning play services to:
```
  ext.play_services_version = '10.2.1'
```
Relevant section of the ./gradlew dependencies output from my app:
```
+--- com.siftscience:sift-android:0.9.2
|    +--- com.fasterxml.jackson.core:jackson-core:2.3.3
|    +--- com.fasterxml.jackson.core:jackson-databind:2.3.3
|    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.3.0 -> 2.3.3
|    |    \--- com.fasterxml.jackson.core:jackson-core:2.3.3
|    +--- com.fasterxml.jackson.core:jackson-annotations:2.3.3
|    +--- javax.validation:validation-api:1.1.0.Final
|    +--- javax.annotation:javax.annotation-api:1.2
|    +--- com.google.guava:guava:19.0 -> 20.0
|    +--- com.squareup.okhttp3:okhttp:3.4.1 -> 3.4.2
|    |    \--- com.squareup.okio:okio:1.9.0 -> 1.11.0
|    +--- org.apache.commons:commons-lang3:3.5
|    +--- com.google.android.gms:play-services-base:[9.8.0,) -> 11.0.0 (*)
|    \--- com.google.android.gms:play-services-location:[9.8.0,) -> 11.0.0 (*)
```

It looks like this was an intentional change in 0ca2337e78aa18752aaf563c680309b3c176b430 but I'd argue it's a mistake.  If you specify 9.8.0 in your library, and I specify 10.2.1 in my app, it'll seamlessly use 10.2.1 for both.  If you specify `[9.8.0,)` it apparently forces everybody up to the latest (which is a notoriously [bad practice](http://blog.danlew.net/2015/09/09/dont-use-dynamic-versions-for-your-dependencies/) on Android).

- I'll get around this for now by using a resolutionStrategy, but that's really unfortunate.  
- a `CHANGELOG.md` would be great so figuring out this sort of thing is easier in the future.

**Comment by garylee1** [comment]
Ah, I can change this back @edenman . There was some a version mismatch which was causing an NPE for customers that were importing an older play-services version.

**Comment by garylee1** [comment]
For context, it was something to this effect:

```
java.lang.AbstractMethodError: abstract method "com.google.android.gms.common.api.Api$zze com.google.android.gms.common.api.Api$zza.zza(android.content.Context, android.os.Looper, com.google.android.gms.common.internal.zzg, java.lang.Object, com.google.android.gms.common.api.GoogleApiClient$ConnectionCallbacks, com.google.android.gms.common.api.GoogleApiClient$OnConnectionFailedListener)"
at com.google.android.gms.common.api.GoogleApiClient$Builder.zza(Unknown Source)
at com.google.android.gms.common.api.GoogleApiClient$Builder.zzuQ(Unknown Source)
at com.google.android.gms.common.api.GoogleApiClient$Builder.build(Unknown Source)
at siftscience.android.AppStateCollector.(AppStateCollector.java:62)
at siftscience.android.Sift.open(Sift.java:57)
at siftscience.android.Sift.open(Sift.java:46)
```

**Comment by edenman** [comment]
Gotcha.  Yeah, if you guys need `9.8.0`, just refer to that and it should work fine.

**Comment by garylee1** [comment]
I think it was a version discrepancy specifically between an old version of `play-services-base` and `play-services-location:9.8.0` that was causing the NPE. I guess instead of coercing the version we can ask customers to bump their `play-services-base` version to at least `9.8.0` if they're using our SDK. Thanks again for the feedback!

**Comment by garylee1** [comment]
this has been addressed by v0.9.3

**Comment by edenman** [comment]
Thanks @garylee1 !
